<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>721</width>
      <height>411</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="imDis">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>719</width>
       <height>409</height>
      </rect>
     </property>
     <widget class="QLabel" name="la_im1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>dummy</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="general">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>general</string>
    </property>
    <widget class="QComboBox" name="cb_SLMmode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="la_xOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x offset:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_xOff">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>600.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="la_yOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y offset:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_yOff">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>290.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="phase">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Phase</string>
    </property>
    <widget class="QLabel" name="la_r">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>r:</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>x:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_yAngle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>45.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_xAngle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="la_y">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_rAngle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>15.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_xPeriod">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-5000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>-3.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_xNA">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.350000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_yNA">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.400000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_yPeriod">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-5000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_rNA">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.350000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_rPeriod">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-5000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>8.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="la_phaseLabels">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>angle      period        NA</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_radGauss">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>radius lens:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_radGauss">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-5000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>150.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="advanced">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Advanced">
     <attribute name="title">
      <string>advanced</string>
     </attribute>
     <widget class="QSpinBox" name="sb_yPix">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>50000</number>
      </property>
      <property name="value">
       <number>600</number>
      </property>
     </widget>
     <widget class="QLabel" name="la_lambda">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>wavelength:</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_xPix">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>x pix:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_xPix">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>50000</number>
      </property>
      <property name="value">
       <number>800</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sb_lambda">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="value">
       <double>0.492000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="la_yPix">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>y pix:</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_fLens">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>f_lens:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsb_fLens">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>9</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <double>50000.000000000000000</double>
      </property>
      <property name="value">
       <double>500.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsb_fObj">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>25</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="la_fObj">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>25</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>f_objective:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsb_refractiveInd">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>1.330000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="la_refractiveInd">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>n:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsb_pixSizeSLM">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>55</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.020000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="la_pixSizeSLM">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>51</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SLM pix:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="advMeelad">
     <attribute name="title">
      <string>Meelad</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="dsb_meelad_dy">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-300.000000000000000</double>
      </property>
      <property name="maximum">
       <double>300.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>30.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsb_meelad_dx">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-400.000000000000000</double>
      </property>
      <property name="maximum">
       <double>400.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>30.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="la_meelad_dy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>dy:</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_meelad_dx">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>dx:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="mask">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>120</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Mask</string>
    </property>
    <widget class="QDoubleSpinBox" name="dsb_rollOff">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="la_radOut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>rad out:</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_rollOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>roll off:</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_radIn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>rad in:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_radIn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>80.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_radius">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>150.000000000000000</double>
     </property>
    </widget>
   </widget>
   <zorder>mask</zorder>
   <zorder>general</zorder>
   <zorder>phase</zorder>
   <zorder>advanced</zorder>
   <zorder>scrollArea</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
